package.json

```diff
@@ -86,7 +86,7 @@
       "require": "./dist/cjs/*.js",
       "default": "./dist/*.js"
     },
-    "./lang/*": {
+    "./dist/lang/*": {
```

Comment:
does this work to resolve both `/lang/pt.js` and `/lang/pt` in a bundler setup? 

also just noticed we should order the `exports` keys from specific to non-specific because the first match gets chosen and stops. basically the lang related should go above the `./*` and `./*.js`